docs/guide/protocols/septop.rst

```diff
@@ -0,0 +1,131 @@
+two angle, and three dihedral restraints. Reference atoms are picked based on different criteria, such as the root mean squared
+fluctuation of the atoms in a short MD simulation, the secondary structure of the protein, and the distance between atoms, based on heuristics from Baumann et al. [2]_.
+
+Partial annihilation scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the :class:`SepTopProtocol <.SepTopProtocol>` the coulombic interactions of the molecules are fully turned off (annihilated) in the respective non-interacting end states.
+The Lennard-Jones interactions are instead decoupled, meaning the intermolecular interactions turned off, keeping the intramolecular Lennard-Jones interactions.
+
+The lambda schedule
+~~~~~~~~~~~~~~~~~~~
+
+Molecular interactions are modified along an alchemical path using a discrete set of lambda windows.
+For the transformation of ligand A to ligand B in the binding site, the following steps are carried out, starting with ligand A being fully interacting in the binding site while ligand B is decoupled.
+
+1. Insert the non-interacting dummy ligand B into the binding site and restrain it using orientational restraints. The contribution of the restraints is calculated analytically.
+2. Turn on the van der Waals (vdW) interactions of ligand B while also turning on orientational restraints on ligand A.
+3. Turn on the electrostatic interactions of ligand B while at the same time turning off the electrostatics of ligand A.
+4. Turn off vdW interactions of ligand A while simultaneously releasing restraints on ligand B.
+5. Release the restraints of the now dummy ligand A analytically and transfer the ligand into the solvent.
+
+The lambda schedule in the solvent phase is similar to the one in the complex, except that a single harmonic distance restraint is
+applied between the respective central atom in the two ligands to keep the ligands apart while doing the alchemical transformation.
+A soft-core potential from Beutler et al. [3]_ is applied to the Lennard-Jones potential to avoid instablilites in intermediate lambda windows.
+The lambda schedule is defined in the ``lambda_settings`` objects ``lambda_elec_A``, ``lambda_elec_B``,  ``lambda_vdw_A``, ``lambda_vdw_B``,
+``lambda_restraints_A``, and ``lambda_restraints_B``.
+
+Simulation overview
+~~~~~~~~~~~~~~~~~~~
+
+The :class:`.ProtocolDAG` of the :class:`SepTopProtocol <.SepTopProtocol>` contains :class:`.ProtocolUnit`\ s from both the complex and solvent transformations.
+This means that both legs of the thermodynamic cycle are constructed and run sequentially in the same :class:`.ProtocolDAG`. This is different from the :class:`.RelativeHybridTopologyProtocol` where the :class:`.ProtocolDAG` only runs a single leg of a thermodynamic cycle.
+If multiple ``protocol_repeats`` are run (default: ``protocol_repeats=3``), the :class:`.ProtocolDAG` contains multiple :class:`.ProtocolUnit`\ s of both complex and solvent transformations.
```

Comment:
Unclear from this text if that means that the same restraint will be used for all repeats.